# Event for containing a single 

# Timestamp
Header header

# Flag for whether or not the event has been sequenced
bool is_sequenced

# Gamma-ray interactions
GammaRayInteraction int1
GammaRayInteraction int2
